models/DataObject/ClassDefinition/Data/Video.php

```diff
@@ -70,6 +82,20 @@ class Video extends Data implements ResourcePersistenceAwareInterface, QueryReso
      */
     public $columnType = 'text';
 
+    /**
+     * @internal
+     *
+     * @var array
+     */
+    public $allowedTypes;
+
+    /**
+     * @internal
+     *
+     * @var array
+     */
+    public $videoTypes = self::ALLOWED_TYPES;
```

Comment:
This feels like a workaround, probably it's better to use `FieldDefinitionEnrichmentInterface`
See https://github.com/pimcore/pimcore/blob/6245efe3be3d64ca5f3369a8472533f98d73043e/models/DataObject/ClassDefinition/Data/Multiselect.php#L713 for an example.